EventFilter/SiStripRawToDigi/plugins/SiStripDigiToRawModule.cc

```diff
@@ -21,10 +23,12 @@ namespace sistrip {
   DigiToRawModule::DigiToRawModule( const edm::ParameterSet& pset ) :
     inputModuleLabel_( pset.getParameter<std::string>( "InputModuleLabel" ) ),
     inputDigiLabel_( pset.getParameter<std::string>( "InputDigiLabel" ) ),
+    copyBufferHeader_(pset.getUntrackedParameter<bool>("CopyBufferHeader",false)),
     mode_( fedReadoutModeFromString(pset.getParameter<std::string>( "FedReadoutMode" ))),
     rawdigi_( false ),
     digiToRaw_(0),
-    eventCounter_(0)
+    eventCounter_(0),
+    rawDataTag_(pset.getUntrackedParameter<edm::InputTag>("RawDataTag",edm::InputTag("rawDataCollector","")))
```

Comment:
Please comment or follow up: I think that RawDataTag and CopyBufferHeader should be tracked parameters